### ISSUE TITLE
Sorry an unexepected error occured with v1.3.171 unstable

### ISSUE THREAD

**Issue by badabing2005** [greenshot/greenshot#295]
Just installed v1.3.171 and got this error on launch.

```
Software version: 1.3.171 - 2bbaa4a9a7 (64 bit)
.NET runtime version: 4.0.30319.42000+
Time: 2021-03-21 19:01:44 -04:00
OS: Windows 10 2009 x64 build 19042
GDI object count: 94
User object count: 50

Exception: System.NullReferenceException
Message: Object reference not set to an instance of an object.

Stack:
   at GreenshotPlugin.Core.CaptureDetails.HasDestination(String designation) in D:\a\1\s\GreenshotPlugin\Core\CaptureDetails.cs:line 111
   at Greenshot.Helpers.CaptureHelper.HandleCapture() in D:\a\1\s\Greenshot\Helpers\CaptureHelper.cs:line 663
   at Greenshot.Helpers.CaptureHelper.MakeCapture() in D:\a\1\s\Greenshot\Helpers\CaptureHelper.cs:line 416
   at Greenshot.Helpers.CaptureHelper.CaptureFile(String filename) in D:\a\1\s\Greenshot\Helpers\CaptureHelper.cs:line 167
   at Greenshot.Forms.MainForm.CaptureFile() in D:\a\1\s\Greenshot\Forms\MainForm.cs:line 728
   at System.Windows.Forms.Control.InvokeMarshaledCallbackHelper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Forms.Control.InvokeMarshaledCallback(ThreadMethodEntry tme)
   at System.Windows.Forms.Control.InvokeMarshaledCallbacks()

Configuration dump:
```

It was working fine with 1.3.157 before the upgrade.
I installed it to address this issue.
https://github.com/greenshot/greenshot/issues/279

Thanks

**Comment by badabing2005** [comment]
Correction, Greenshot launches ok, however when I capture a screenshot, nothing shows up, and if I attempt to open a previously captured screenshot, I get the error above.

**Comment by Lakritzator** [comment]
Should be fixed in 1.3.172 which is coming up right now.

**Comment by badabing2005** [comment]
Thanks, can confirm that 1.3.172 indeed fixes the issue.
By the way I noticed that the OCR plugin was missing from both 1.3.171 and 1.3.172 builds, not sure if it is by design or not, just mentioning it in case it was unintentional.

Thanks a bunch
BB
